asistente_ladm_col/gui/create_administrative_source_cadastre_wizard.py

```diff
@@ -72,10 +85,23 @@ def finished_dialog(self):
         self.save_settings()
 
         if self.rad_refactor.isChecked():
+
+            if self.cbo_mapping.currentText() is not "":
```

Comment:
Do not compare values using the `is` keyword... In this case, `if not self.cbo_mapping.currentText()`: should be enough.